src/main/java/com/atlassian/shele/shelePlugin/ao/ProjectEntityService.java

```diff
@@ -1,36 +1,42 @@
 package com.atlassian.shele.shelePlugin.ao;
 
 import com.atlassian.activeobjects.external.ActiveObjects;
-import com.atlassian.jira.component.ComponentAccessor;
-import com.atlassian.jira.event.type.EventType;
 import com.atlassian.plugin.spring.scanner.annotation.imports.ComponentImport;
 import net.java.ao.Query;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
 
 import static com.google.gson.internal.$Gson$Preconditions.checkNotNull;
 
-@Component
+@Service
 public class ProjectEntityService {
     private final ActiveObjects activeObjects;
-    private final MapIssueMapper mapper;
 
-    @Autowired
-    public ProjectEntityService(@ComponentImport ActiveObjects activeObjects, MapIssueMapper mapper) {
+
+
+    public ProjectEntityService(@ComponentImport ActiveObjects activeObjects) {
         this.activeObjects = checkNotNull(activeObjects);
-        this.mapper= mapper;
+
     }
 
-    public ProjectEntity getEntity(){
-        ProjectEntity entity= activeObjects.create(ProjectEntity.class);
-        entity.setProject(String.valueOf(ComponentAccessor.getProjectManager().getProjects()));
-        entity.setEventTypeId(ComponentAccessor.getEventTypeManager().getEventTypes().stream().map(EventType::getId).toString());
-        entity.setIssueStatus(ComponentAccessor.getWorkflowManager().getDefaultWorkflow().toString());
-        entity.save();
+    public ProjectEntity saveEntity(ProjectDTO dto) {
+        List<String> list = dto.getProject();
+        List<Long> longs = dto.getEventTypeIds();
+        ProjectEntity entity = activeObjects.create(ProjectEntity.class);
+        for (String s : list) {
+            entity.setProject(s);
+            entity.setEventTypeId(longs.toString());
+            entity.save();
```

Comment:
also, it looks like you always just update the project entity value to the last one from the whole list of values but not the each one